src/main/java/pl/betoncraft/betonquest/Journal.java

```diff
@@ -77,7 +79,12 @@ public static boolean isJournal(final String playerID, final ItemStack item) {
         return item.getType().equals(Material.WRITTEN_BOOK) && ((BookMeta) item.getItemMeta()).hasTitle()
                 && ((BookMeta) item.getItemMeta()).getTitle().equals(Config.getMessage(playerLang, "journal_title"))
                 && item.getItemMeta().hasLore()
-                && item.getItemMeta().getLore().contains(Config.getMessage(playerLang, "journal_lore"));
+                && Objects.equals(item.getItemMeta().getLore(), getJournalLore(playerLang));
+    }
+
+    @NotNull
+    private static List<String> getJournalLore(final String lang) {
+        return new ArrayList<>(Arrays.asList(Utils.format(Config.getMessage(lang, "journal_lore")).split("\n")));
```

Comment:
Why wrap everything in a ArrayList? Arrays.asList already returns a List implementation.

Reply:
True, that was from a some refactors, fixed